src/common/utils.lua

```diff
@@ -333,4 +333,42 @@ function utils.affiliationsForAddress(address, ants)
 	return affiliations
 end
 
+--- @param versionString string
+function utils.validateSemver(versionString)
+	assert(type(versionString) == "string", "Version must be a string")
+
+	-- Reject versions with build metadata (anything after +)
+	if versionString:find("%+") then
```

Comment:
Why would this happen? This seems like a very specific negative case. Can we just focus on the positive cases?

Reply:
Will remove - stepping away from semver to iterative versions